src/Aquifer.Data/Migrations/20240828185812_AddIpAddressDataTable.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Aquifer.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddIpAddressDataTable : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
```

Comment:
We should double check we've deployed Public API and Jobs to prod before this gets run.